### PR TITLE
[Cias30-3154] verify participant in epic

### PR DESCRIPTION
**Pull request by WiktoriaRozanska**
## Related tasks
- [CIAS-3154](https://htdevelopers.atlassian.net/browse/CIAS30-3154)

## What's new?
- service to check if the patient exists in third-party database
- service to fetch all appointments assigned to selected patient
- adjusted verifying endpoint and its service

### CODE REVIEW

File: app/services/api/epic_on_fhir/patient_verification.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          name: 'onlyCertainMatches',
+          valueBoolean: 'true'
+        }
+      ]
+    }.to_json
+  end
+
+  def not_found_condition(response)
+    response[:total] != 1
+  end
```
Comment (msniecHTD) [inline app/services/api/epic_on_fhir/patient_verification.rb:72]: What do you think about having two different methods for when no results are found and when there are more than 1 result

File: config/locales/en.yml [hunk 2]
```diff
@@ -319,3 +316,11 @@ en:
     intervention_conversation_transcripts_ready:
       subject: "Your transcripts are ready"
       body: "Your conversation transcripts from intervention <span style='color:#C866EA;'>%{intervention_name}</span> are ready."
+  epic_on_fhir:
+    error:
+      authentication: "Authorization process failed. Please contact our support team so that we can assist you in resolving this matter"
+      unexpected_error: "Third party tool return unexpected response. Please contact our support team"
+      patient:
+        not_found: "We could not clearly identify you or associated data in out third party tool. Please contact our support team"
```
Comment (msniecHTD) [inline config/locales/en.yml:324]:
```suggestion
        not_found: "We could not clearly identify you or associated data in third party tool. Please contact our support team"
```

**Review by msniecHTD — APPROVED** [review]
Some minor coments

File: app/services/api/epic_on_fhir/patient_verification.rb (hunk 1 above)
Comment (WiktoriaRozanska) [reply app/services/api/epic_on_fhir/patient_verification.rb:72]: I don't think we need it. As a BE system, we can only use `onlyCertainMatches` so situations when we have more than one is equal the same as we have zero records

**Pull request merged**